src/lib/command_line.ml

```diff
@@ -125,9 +128,9 @@ let pipeline ~biokepi_machine ?work_directory =
     (`With_varscan with_varscan)
     (`With_somaticsniper with_somaticsniper)
     (`Bedfile bedfile)
-    (`Normal_json normal_json_file)
-    (`Tumor_json tumor_json_file)
-    (`Rna_json rna_json_file)
+    (`Normal_json normal_json_files)
```

Comment:
Yeah; each arg is commented with its own docstring (e.g. https://github.com/hammerlab/epidisco/blob/master/src/lib/command_line.ml#L205). These tags are just a way of making sure the types are safer (i.e. a `string` becomes `Mailgun_api_key string`)